app/manifest/v3/_base.json

```diff
@@ -61,6 +62,7 @@
   "name": "__MSG_appName__",
   "permissions": [
     "storage",
+    "scripting",
```

Comment:
We've generally been wary of adding new permissions because if they introduce a user-facing warning, it will disable the extension on update and re-prompt the user to agree to the permissions. But [I tested this locally using these steps](https://developer.chrome.com/docs/extensions/mv2/permission_warnings/#view_warnings), and it looks like there is no warning for this at the moment. So... maybe this is OK.

Reply:
I'm surprised host permissions didn't trigger anything. The added permissions seem the minimal set necessary for maintaining the functionality of window.ethereum. I believe we don't have a different option for mv3 unless we find a way for the content script in MAIN world to communicate directly.